config/parse_vault_mount.go

```diff
@@ -16,7 +15,7 @@ func (c *Config) processVaultMounts(list *ast.ObjectList, environment *Environme
 	for _, mountAST := range list.Items {
 		x := mountAST.Val.(*ast.ObjectType).List
 
-		valid := []string{"config", "role", "type", "path"}
+		valid := []string{"config", "role", "type", "path", "maxleasettl"}
```

Comment:
oh, and add `default_lease_ttl` and `force_no_cache` that should cover all 3 options `MountConfigInput` covers - if you don't got time, i can do that